comms/lighthouse/src/peerjs-server/messageHandler/handlersRegistry.ts

```diff
@@ -17,7 +17,7 @@ export class HandlersRegistry implements IHandlersRegistry {
     this.handlers.set(messageType, handler)
   }
 
-  public handle(client: IClient | undefined, message: IMessage): boolean {
+  public handle(client: IClient | undefined, message: IMessage): boolean | Promise<boolean> {
```

Comment:
Is there a reason to keep `bool | Promise<bool>` instead of simply `Promise<bool>`? in the later we can take advantage of linters to warn us about

```ts
const a: bool | Promise<bool> = fn()

// not handled by linter, may always return "true" because a is a Promise
if(a) {...}
```

Reply:
Hmm, the only reason I made it so it could return boolean | Promise was because it seemed like it would limit the ripple effect of the change. But it is true that it won't be so hard to go full promises, so I'll do that.